_data/versions.yml

```diff
@@ -107,6 +107,84 @@ v3.1:
       url: http://git.openstack.org/cgit/openstack/networking-calico/commit/?h=2.0.0
 
 v3.0:
+- title: v3.0.4
+  note: |
+    21 March 2018
+
+    > **Important**: [Review the documentation](https://docs.projectcalico.org/v3.0/getting-started/kubernetes/upgrade/) carefully before attempting an upgrade.
```

Comment:
Are we now supposed to have the text in _versions.yml and in the MD file? I guess I didn't expect it.

Reply:
Bah, no. I'm not sure why this is still here...